sdk_contrib/aws_v1_support/build.gradle

```diff
@@ -12,12 +12,9 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
-    api project(':opentelemetry-api'),
-            project(':opentelemetry-sdk')
+    api project(':opentelemetry-api')
+    api project(':opentelemetry-sdk')
 
-    implementation libraries.guava
-
-    implementation 'com.amazonaws:aws-xray-recorder-sdk-core:2.4.0'
     implementation 'com.amazonaws:aws-java-sdk-core:1.11.701'
```

Comment:
Can be done in on implementation statement

Reply:
Sure - I took the opposite approach of making them consistent but went with one statement. But I'd argue it's better style to use multiple statements, there's a significant readability loss when the dependencies for the same configuration don't line up vertically IMO. Food for thought.